rest-api-spec/src/main/resources/rest-api-spec/api/cat.allocation.json

```diff
@@ -19,7 +19,7 @@
         "bytes": {
           "type": "enum",
           "description" : "The unit in which to display byte values",
-          "options": [ "b", "k", "m", "g" ]
+          "options": [ "b", "k", "kb", "m", "mb", "g", "gb", "t", "tb", "p", "pb" ]
```

Comment:
can you also update cat.fielddata.json, cat.indices.json and cat.recovery.json? (These were returned by grepping `"bytes": {`)